### PR TITLE
Python: Fix stdlib sinks in LogInjection query

### PR DESCRIPTION
**Pull request by RasmusWL**

### CODE REVIEW

File: python/ql/lib/semmle/python/security/dataflow/LogInjectionCustomizations.qll [hunk 1]
```diff
@@ -47,7 +47,35 @@ module LogInjection {
...
-    LoggingAsSink() { this = any(Logging write).getAnInput() }
+    LoggingAsSink() {
+      this = any(Logging write).getAnInput() and
+      // since the inner implementation of the `logging.Logger.warn` function is
+      // ```py
+      // class Logger
+      //     def warn(self, msg, *args, **kwargs):
+      //     warnings.warn("The 'warn' method is deprecated, "
+      //         "use 'warning' instead", DeprecationWarning, 2)
+      //     self.warning(msg, *args, **kwargs)
```
Comment (yoff) [inline python/ql/lib/semmle/python/security/dataflow/LogInjectionCustomizations.qll:58]:
```suggestion
      // class Logger:
      //     def warn(self, msg, *args, **kwargs):
      //         warnings.warn("The 'warn' method is deprecated, "
      //             "use 'warning' instead", DeprecationWarning, 2)
      //         self.warning(msg, *args, **kwargs)
```

**Review by yoff — CHANGES_REQUESTED** [review]
Small tweak requested, the example was a bit hard to read.
Seems like a fine solution.

File: python/ql/lib/semmle/python/security/dataflow/LogInjectionCustomizations.qll (hunk 1 above)
Comment (RasmusWL) [reply python/ql/lib/semmle/python/security/dataflow/LogInjectionCustomizations.qll:58]: oh, I must have messed that up when I copy-pasted :see_no_evil: thanks :heart:

**Review by yoff — APPROVED** [review]
LGTM

**Pull request merged**